src/devices/GoPiGo3/Models/GrooveSensors.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.GoPiGo3.Models
+{
+    public class GrooveSensor
+    {
+        /// <summary>
+        /// Constructor for the groove sensor part of the main GoPiGo3 class
+        /// </summary>
+        /// <param name="groovePort">The Groove port, either Groove1 or Groove2</param>
+        public GrooveSensor(GroovePort groovePort)
+        {
+            if (!((groovePort != GroovePort.Groove1) || (groovePort != GroovePort.Groove2)))
```

Comment:
Are we sure this is the right condition?

Based on the exception message, shouldn't it be:
```C#
if ((groovePort != GroovePort.Groove1) && (groovePort != GroovePort.Groove2))
```

Reply:
Correct. Your is more clear.